toptal/picasso
# [FX-5348] Update Avatar component to follow IMF guidelines

[FX-5348]

### Description

This PR adds a new prop called `emblem` to the Avatar and AvatarGroup components so that the Toptal logo can be displayed if needed in the lower left corner. It also removes the `variant` prop that was used to display the avatar in a `portrait` or `landscape` view.

**_Note_**: The dimensions for the emblem are set according to the Figma [designs](https://www.figma.com/design/5SCTOPrCDcHuk5We091GBn/branch/bUFkua511v5UOOpn08KSZO/Product-Library?node-id=340-14433&t=Gj4q8N9lVNS4syvg-0).

### How to test


- [Temploy](https://picasso.toptal.net/fx-5348-update-avatar-component-to-follow-imf-guidelines)
- Check the [production](https://picasso.toptal.net/?path=/story/components-avatar--avatar) and compare it with the temploy.

### Screenshots

| Before.                                 | After.                                  |
| --------------------------------------- | --------------------------------------- |
| ![image](https://github.com/toptal/picasso/assets/100089376/e4c34eab-8cd2-4c89-807b-7d4e5a34e922) | ![image](https://github.com/toptal/picasso/assets/100089376/17135cff-2711-4d9a-9b57-543ee3101904) |

### Development checks

- ~Read [CONTRIBUTING.md](https://github.com/toptal/picasso/blob/master/CONTRIBUTING.md) and [Component API principles](https://github.com/toptal/picasso/blob/master/docs/contribution/component-api.md)~
- ~Annotate all `props` in component with documentation~
- ~Ensure that deployed demo has expected results and good examples~
- ~Ensure the changed/created components have not caused accessibility issues. [How to use accessibility plugin in storybook](https://github.com/toptal/picasso/blob/master/docs/contribution/accessibility.md).~

**Breaking change**

- ~codemod is created and showcased in the changeset~
- ~test alpha package of Picasso in StaffPortal~

> All **_development checks_** should be done and set checked to pass the
> **GitHub Bot: TODOLess** action

<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run package:alpha-release` - Release alpha version
- `@toptal-anvil ping reviewers` - Ping FX team for review

</details>

<details>
<summary>PR Review Guidelines</summary>
<br />

#### When to approve? ✅

**You are OK** with merging this PR and

1. You have no extra requests.
2. You have optional requests.
   1. Add `nit:` to your comment. (ex. `nit: I'd rename this variable from makeCircle to getCircle`)

#### When to request changes? ❌

**You are not OK** with merging this PR because

1. Something is broken after the changes.
2. Acceptance criteria is not reached.
3. Code is dirty.

#### When to comment (neither ✅ nor ❌)

**You want your comments to be addressed** before merging this PR in cases like:

1. There are leftovers like unnecessary logs, comments, etc.
2. You have an opinionated comment regarding the code that requires a discussion.
3. You have questions.

#### How to handle the comments?

1. An owner of a comment is the only one who can resolve it.
2. An owner of a comment must resolve it when it's addressed.
3. A PR owner must reply with ✅ when a comment is addressed.

</details>


[FX-5348]: https://toptal-core.atlassian.net/browse/FX-5348?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
